src/goInstallTools.ts

```diff
@@ -148,16 +144,19 @@ export function installTools(missing: Tool[], goVersion: GoVersion): Promise<voi
 			envForTools['GO111MODULE'] = 'on';
 		}
 
+		// Write a temporary go.mod file. This shouldn't affect anything if we've explicitly disabled modules.
+		const tmpGoModFile = path.join(toolsTmpDir, 'go.mod');
+		fs.writeFileSync(tmpGoModFile, 'module tools');
```

Comment:
With this, we are creating a new go.mod file for each tool being installed right?
In that case, should we move it inside the `else` block of the `if (modulesOffForTool)` check a few lines above?

Reply:
Also, did you check if `fs.writeFileSync` fails gracefully if the file already exists?

Reply:
It shouldn't matter if we write the file even if modules are off, but I moved the check in case. `fs.writeFileSync` replaces the file if it already exists.